install-unix.sh

```diff
@@ -42,11 +42,11 @@ fi
 echo installling event_name=$1
 
 vm_cmd="cd ~/uber/ && ./run-simple-deploy.sh $1 && rm -f ./run-simple-deploy.sh"
-git clone https://github.com/magfest/ubersystem-deploy
+test -d ubersystem-deploy || git clone https://github.com/magfest/ubersystem-deploy
 cd ubersystem-deploy
```

Comment:
or, we could just kill that for now too, the default error message is cryptic but does the right thing.  it would be cool to have the more detailed description.  if you didn't feel like modifying it right now, you could create an issue like 'display better error message + instructions when ubersystem-deploy already exists'